src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c

```diff
@@ -287,8 +287,8 @@ static int ospi_versal_remove(struct platform_device *pdev)
 	platform_set_drvdata(pdev, NULL);
 	xocl_drvinst_free(hdl);
 
-done:
 	OV_INFO(ov, "return: %d", ret);
+done:
```

Comment:
It is OK, but I don't see much value to use a goto in this case. Returning -EINVAL at line 279 will be just fine